Sources/StreamChat/Controllers/CurrentUserController/CurrentUserController.swift

```diff
@@ -257,6 +257,25 @@ public extension CurrentChatUserController {
             }
         }
     }
+    
+    /// Get all blocked users.
+    ///
+    /// - Parameter completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    ///
+    func loadBlockedUsers(completion: @escaping (Result<[BlockedUserDetails], Error>) -> Void) {
+        currentUserUpdater.loadBlockedUsers {
+            switch $0 {
+            case let .success(blockedUsers):
+                self.callback {
+                    completion(.success(blockedUsers))
+                }
+            case let .failure(error):
+                self.callback {
+                    completion(.failure(error))
+                }
+            }
```

Comment:
Seems like this could be just
```
self.callback {
completion($0)
}
```
since Result types match

Reply:
Done 👍